### PR TITLE
Implement a first user panel and small fixes

### PR DESCRIPTION
**Pull request by yamabiiko**
A template for user panel with some functionalities has been implementd, but most voices are still empty, also it lacks of a panel for admin (it will be provided soon)
Small fixes:
- remove rack-cors since we don't need to use the siamining site anymore
- better translation in italian ("criptazione" -> "crittografia")

### CODE REVIEW

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+
+
+#settings {
+  width: 250px;
+  padding: 8px 10px;
+  display: inline-block;
+  border-radius: 0.25rem;
+  margin: 30px;
+  font-size: 1rem;
+}
```
Comment (FiloSpaTeam) [inline app/assets/stylesheets/users.scss]: Like the other SCSS files the last brackets should go to the last row of the directive :+1:

File: app/assets/stylesheets/users.scss [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
+a:hover {
+  background-color: rgba(0, 0, 0, 0.03);
+  color: #212529;
+}
+
+a:active {
+  background-color: rgba(0, 0, 0, 0.2);
+}
+
+.black-font {
```
Comment (FiloSpaTeam) [inline app/assets/stylesheets/users.scss]: If you check Bootstrap doc you can find a class for dark font :+1:

File: app/assets/stylesheets/users.scss [hunk 3]
```diff
@@ -1,3 +1,40 @@
 // Place all the styles related to the users controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+#settings {
+  width: 250px;
+  padding: 8px 10px;
+  display: inline-block;
+  border-radius: 0.25rem;
```
Comment (FiloSpaTeam) [inline app/assets/stylesheets/users.scss]: If you check Bootstrap class you can find a class that round corners :+1:

**Comment by yamabiiko** [comment]
Changes done.

**Pull request merged**